Simbody/include/simbody/internal/GeneralForceSubsystem.h

```diff
@@ -85,6 +85,22 @@ class SimTK_SIMBODY_EXPORT GeneralForceSubsystem : public ForceSubsystem {
     be explicitly enabled. **/
     void setForceIsDisabled
        (State& state, ForceIndex index, bool shouldBeDisabled) const;
+       
+    /** Set the number of threads that the GeneralForceSubsystem can use to
+    calculate computationally expensive forces (that have the
+    shouldBeParallelIfPossible() method overridden). By default, the
+    number of threads is the number of total processors (including hyperthreads)
+    on the machine.
+    
+    Note: This method should only be called before or after force contributions
+    are calculated (Stage::Dynamics). This method may NOT be called while
+    realizing Stage::Dynaimcs.**/
```

Comment:
This comment is confusing: "only before or after" logically means "any time". Also there is a typo.